integrations/mixpanel/test/index.test.js

```diff
@@ -556,5 +556,27 @@ describe('Mixpanel', function() {
         analytics.called(window.mixpanel.alias, 'new', 'old');
       });
     });
+
+    describe('#group', function() {
+      beforeEach(function() {
+        analytics.stub(window.mixpanel, 'group');
+        analytics.stub(window.mixpanel, 'set_group');
+      });
+
+      it('should not call set_group if groupId is passed or null/undefined/empty', function() {
+        analytics.group('');
+        analytics.didNotCall(window.mixpanel.set_group);
```

Comment:
Could we also check we don't call `window.mixpanel.group`?

Reply:
> Could we also check we don't call `window.mixpanel.group`?

Fixed